internal/builders/golang/build.go

```diff
@@ -44,8 +44,8 @@ func (*Builder) WithDefaults(build config.Build) config.Build {
 	if len(build.Goarm) == 0 {
 		build.Goarm = []string{"6"}
 	}
-	if build.Ldflags == "" {
-		build.Ldflags = "-s -w -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.date={{.Date}}"
+	if len(build.Ldflags) == 0 {
+		build.Ldflags = append(build.Ldflags, "-s -w -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.date={{.Date}}")
```

Comment:
it is not needed to call `append` here right? just `build.Ldflags = []string{"-s -w -X main.version={{.Version}} -X main.commit={{.Commit}} -X main.date={{.Date}}"}`.

I think it is more clear, as we are not really appending to an array with items...